linkerd/datadog_checks/linkerd/linkerd.py

```diff
@@ -32,5 +32,6 @@ def process(self, scraper_config, metric_transformers=None):
             super(LinkerdCheck, self).process(scraper_config, metric_transformers=metric_transformers)
         except Exception:
             self.gauge(self.HEALTH_METRIC, 1, tags=tags)
+            raise
```

Comment:
Can we also log a message?

Reply:
Doesn't `raise` alone raises the same exception?
Reading this: https://docs.python.org/3/tutorial/errors.html#handling-exceptions I thought it would. I wanted to keep the exact same exception to display in the status.